src/lambda/commands/showErrorDetails.ts

```diff
@@ -10,7 +10,7 @@ import _ = require('lodash')
 import * as vscode from 'vscode'
```

Comment:
isn't this whole file generic now? there doesn't appear to be an lambda specific logic in it - but it's still in the lambda src folder...

Reply:
I'm on the fence about this one...the only place we're using this webview logic is Lambda.

Reply:
Actually, never mind, looks like we use this for all error nodes.

Reply:
✔️